components/brave_wallet/common/common_utils.h

```diff
@@ -72,6 +72,9 @@ std::string GetNetworkForCardanoKeyring(const mojom::KeyringId& keyring_id);
 std::string GetNetworkForCardanoAccount(const mojom::AccountIdPtr& account_id);
 
 bool IsPolkadotKeyring(mojom::KeyringId keyring_id);
+bool IsPolkadotNetwork(std::string_view network_id);
+std::string GetNetworkForPolkadotKeyring(const mojom::KeyringId& keyring_id);
+std::string GetNetworkForPolkadotAccount(const mojom::AccountIdPtr& account_id);
```

Comment:
`GetNetworkForPolkadotKeyring`
`GetNetworkForPolkadotAccount`
need unit tests

Reply:
This is done!